views/pages/AccountPage/AccountPage.css

```diff
@@ -0,0 +1,16 @@
+.AccountPage {
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  padding: 70px 125px 70px 24px;
```

Comment:
what is the large right padding for?

Reply:
The parent div is on flex-grow 1 so it's great that it can cover the entire right side of the screen. On the designs of this page the right side is much more further away than the left side. 

<img width="711" alt="Screenshot 2023-11-15 at 3 13 36 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/67d38fc2-bf90-4d7d-95ae-fca7dce54bb0">